packages/pwashortcut.rb

```diff
@@ -0,0 +1,169 @@
+export PREFIX=$CREW_PREFIX/lib/pwa
+export FLASK_APP=$PREFIX/main.py
+export FLASK_ENV=development
+pkill flask
+if [[ $1 = '-h' ]]; then
+    echo  '
+===================================
+    Shortcut Server Starter
+  -s (Default option) Start shortcut server
+  -n (App Name)       Make a new shortcut
+  -h                  Show this message
+  -p                  Set py script path
+  -y                  Set PYTHONPATH
+  -f                  Pass option to Flask
+==================================='
+elif [[ $1 = '-n' ]]; then
+    mkdir $PREFIX/templates
+    mkdir $PREFIX/$2
+    mkdir $PREFIX/$2/templates
+    cp -r $tools/* $PREFIX/$2/templates
+    cp -r $tools/* $PREFIX/templates
+    mv $PREFIX/templates/manifest.json.bak $PREFIX/templates/manifest.json
+    mv $PREFIX/templates/installer.html.bak $PREFIX/templates/installer.html
+    mv $PREFIX/$2/templates/manifest.json.bak $PREFIX/$2/templates/manifest.json
+    mv $PREFIX/$2/templates/installer.html.bak $PREFIX/$2/templates/installer.html
+    sed -i \"s/linuxapp/${2^}/g\" $PREFIX/$2/templates/manifest.json
+    sed -i \"s/unixapp/$2/g\" $PREFIX/$2/templates/manifest.json
+    #######################
+    echo \" \" >> $FLASK_APP
+    echo \"@app.route('/$2/$2.app')\" >> $FLASK_APP
+    echo \"def $2():\" >> $FLASK_APP
+    echo \"    os.system('$2')\" >> $FLASK_APP
+    echo \"    return 'You can close this window now'\" >> $FLASK_APP
+    echo \"@app.route('/$2/')\" >> $FLASK_APP
+    echo \"def installer_$2():\" >> $FLASK_APP
+    echo \"    return render_template('installer.html')\" >> $FLASK_APP
+    echo \"@app.route('/$2/<path:path>')\" >> $FLASK_APP
+    echo \"def $2_path(path):\" >> $FLASK_APP
+    echo \"    if path.endswith(('brew.png', 'manifest.json')):\" >> $FLASK_APP
+    echo \"        return send_from_directory('$2/templates', path)\" >> $FLASK_APP
+    echo \" \" >> $FLASK_APP
+    #######################
+    echo \"Shortcut for ${2^} deployed!\"
+    echo \"Wait for server start and go to localhost:5000/$2/ for installing shortcut.\"
+    sleep 2
+    flask run
+elif [[ $1 = '-s' ]]; then
+    flask run
+else
+    while getopts \"p:y:f:*\" arg; do
+        OPTARG=$(echo ${OPTARG} | sed 's/\=//')
+        RUN=NO
+        case ${arg} in
+           p)
+               echo $OPTARG
+               export FLASK_APP=$OPTARG
+               ;;
+           y)
+               echo $OPTARG
+               export PYTHONPATH=$OPTARG
+               ;;
+           f)  
+               RUN=YES
+               flask run $OPTARG
+               ;;
+           *)  echo \"Error: unknown option '$arg'\"
+               ;;
+        esac
+    done
+    if [[ $RUN != YES ]]; then
+      flask run
+    fi
+fi
+EOF"
+    #########################################
+  end
+
+  def self.install
+    FileUtils.mkdir "#{CREW_PREFIX}/lib/pwa/" unless File.exists? "#{CREW_PREFIX}/lib/pwa/"
+    FileUtils.mkdir "#{CREW_PREFIX}/lib/pwa/tools/" unless File.exists? "#{CREW_PREFIX}/lib/pwa/tools/"
```

Comment:
The 2 lines above could be replaced by one line:
```
FileUtils.mkdir_p "#{CREW_PREFIX}/lib/pwa/tools"
```
No need to include the `unless File.exists? ...` since I doubt any other packages are using this path...yet.